src/features/auth/api.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+const BASE_URL = 'https://comic-dance-club.herokuapp.com';
+// const BASE_URL = 'https://comic-dance-club.herokuapp.com';
```

Comment:
- [ ] Don't forget to remove this when you are done.

Reply:
Done!